src/coalgebras/itreeTauScript.sml

```diff
@@ -1026,6 +1038,113 @@ Proof
   metis_tac[itree_bind_resp_t_wbisim, itree_bind_resp_k_wbisim, itree_wbisim_trans]
 QED
 
+Theorem itree_iter_ret_tau_wbisim:
```

Comment:
If you can't ever see users wanting to use this result outside of this script file, then may add a `[local]` annotation to this one.

Reply:
in what situation should I use `[local]` vs `Triviality`?

Reply:
They’re equivalent.  Use whichever you find most appealing aesthetically.